README.md

```diff
@@ -1 +1,48 @@
-# todolist
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books
+
```

Comment:
- Please, kindly update the project name to be the name of your current project to make your readme more professional.